### PR TITLE
Logos Hyde and Velum

### PR DESCRIPTION
**Pull request by romanogit**

### CODE REVIEW

File: server/game/Card.js [hunk 1]
```diff
@@ -755,4 +755,6 @@ class Card extends EffectSource {
     }
 }
 
+Card.HOUSES = ['brobnar', 'dis', 'logos', 'mars', 'sanctum', 'saurian', 'shadows', 'staralliance', 'untamed'];
```
Comment (cryogen) [inline server/game/Card.js]: This probably doesn't belong here - let's move it to a separate constants file if we don't have one already.  It's ok for it to be split client/server as at some point they will become separate projects

Comment (romanogit) [reply server/game/Card.js]: Done

File: client/constants.js [hunk 2]
```diff
@@ -2,3 +2,8 @@ export const ItemTypes = {
     CARD: 'card',
     PANEL: 'panel'
 };
+
+export const Constants = {
+    HOUSES: ['brobnar', 'dis', 'logos', 'mars', 'sanctum', 'saurian', 'shadows', 'staralliance', 'untamed'],
```
Comment (cryogen) [inline client/constants.js]: CamelCase is fine for constants

File: server/constants.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+class Constants {}
+
+Constants.HOUSES = ['brobnar', 'dis', 'logos', 'mars', 'sanctum', 'saurian', 'shadows', 'staralliance', 'untamed'];
```
Comment (cryogen) [inline server/constants.js]: as above, CamelCase is fine

File: server/game/cards/03-WC/Velum.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
+const Card = require('../../Card.js');
+
+class Velum extends Card {
+    setupCardAbilities(ability) {
+        this.reap({
+            useActionPromptMessage: true,
```
Comment (cryogen) [inline server/game/cards/03-WC/Velum.js]: What's the behaviour without this flag?

Comment (romanogit) [reply server/game/cards/03-WC/Velum.js]: Messages without the flag (read bottom up):

```
player1 uses Velum to archive 2 cards
player1 uses Velum to archive Velum    <----- Velum is not archived. Need to remove this line
player1 uses Velum to reap with Velum
player1 plays Hyde
player1 chooses logos as their active house this turn
player1 does not forge a key.  They have 0 amber.  The current cost is 6 amber
player1: 0 amber player2: 0 amber
player2 readies all their cards
player2 chooses brobnar as their active house this turn
player2 does not forge a key.  They have 0 amber.  The current cost is 6 amber
player1: 0 amber player2: 0 amber
player1 readies all their cards
player1 chooses logos as their active house this turn
player1 does not forge a key.  They have 0 amber.  The current cost is 6 amber
player1 draws 1 cards up to their maximum hand size of 6
player2 draws 6 cards up to their maximum hand size of 6
player1 draws 6 cards up to their maximum hand size of 6
player2 is playing as the Archon: [object Object]
player1 is playing as the Archon: [object Object]
player2 is shuffling their deck
player1 is shuffling their deck
```

Messages with the flag:

```
player1 uses Velum to archive 2 cards
player1 uses Velum to reap with Velum
player1 plays Hyde
player1 chooses logos as their active house this turn
player1 does not forge a key.  They have 0 amber.  The current cost is 6 amber
player1: 0 amber player2: 0 amber
player2 readies all their cards
player2 chooses brobnar as their active house this turn
player2 does not forge a key.  They have 0 amber.  The current cost is 6 amber
player1: 0 amber player2: 0 amber
player1 readies all their cards
player1 chooses logos as their active house this turn
player1 does not forge a key.  They have 0 amber.  The current cost is 6 amber
player1 draws 1 cards up to their maximum hand size of 6
player2 draws 6 cards up to their maximum hand size of 6
player1 draws 6 cards up to their maximum hand size of 6
player2 is playing as the Archon: [object Object]
player1 is playing as the Archon: [object Object]
player2 is shuffling their deck
player1 is shuffling their deck
```

**Pull request merged**